recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch

```diff
@@ -0,0 +1,17 @@
+--- a/CMakeLists.txt	2017-11-24 17:22:06.588688332 +0200
++++ b/CMakeLists.txt	2017-11-24 17:21:51.364715576 +0200
+@@ -34,6 +34,7 @@
+   #   - other distros
+   #   - errors
+   # we define the version to be 2.0.0
++  set(DISTRO_CODENAME "poky")
+   if (${DISTRO_CODENAME} STREQUAL "vivid")
+     set(DBUS_CPP_VERSION_MAJOR 4)
+     set(DBUS_CPP_VERSION_MINOR 3)
+@@ -102,6 +103,5 @@
+ add_subdirectory(examples)
```

Comment:
you probably don't need examples

Reply:
you are right, removing tests, docs and examples from dbus-cpp and process-cpp as well